resources/less/import.less

```diff
@@ -17,11 +17,11 @@
 @import "components/announcement-preview";
 @import "components/archived-issue";
 @import "components/article";
-@import "components/footer";
-@import "components/header";
 @import "components/issue-section";
 @import "components/issue";
 @import "components/journal-desc";
+@import "components/main-footer";
```

Comment:
For me, it makes sense to name those as per template file that they are linked with.

Reply:
I wasn’t sure either actually -- I found it maybe confusing to just only `footer`, `main`, and `header` because you can still have `<header>`s and `<footer>` as descendents of other elements, like `<article>`, so there isn’t necessarily only one header or footer on a page.

Reply:
Actually changing it to `header` and `footer` only - I’m realising that it’s not that confusing after all (it’s singular, so it’s not like they’re stylesheets for all or any types of headers and footers on a page.